scanpy/external/_tools/_palantir.py

```diff
@@ -0,0 +1,286 @@
+"""Run Diffusion maps using the adaptive anisotropic kernel
+"""
+
+from scanpy import logging as logg
+
+
+def palantir( adata ):
```

Comment:
No white space between parentheses and arguments, please respect the PEP440 coding style.